packages/components/src/local/atoms/input-container/types/props.d.ts

```diff
@@ -3,6 +3,14 @@ export default interface PropTypes {
    * The name of the input label
    */
   label?: string
+  /**
+   * The color of the input label
```

Comment:
@zukilover - the story for this uses a value of `common.white`. Could you please add a small note to indicate the expected ways of expressing a color?  I was expecting something like `#fff`.

Reply:
Alright, I will add it to the theme configuration

Reply:
Hello @zukilover . I don't think we need a theme change.  I just wanted a more detailed comment that tells developers what types of string are acceptable.

Reply:
I've added the props description